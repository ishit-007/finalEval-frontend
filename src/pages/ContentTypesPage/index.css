.content-typs-page{
  display: flex;
}