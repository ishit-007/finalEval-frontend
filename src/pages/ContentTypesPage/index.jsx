import React from 'react';
import './index.css';
import { ContentTypes, CollectionTypes } from '../../components';

const ContentTypesPage = () => {
  // const [selectedContentType, setSelectedContentType] = React.useState('CompanyProfile');
  return (
    <div className="content-typs-page">
      <div className="collectionTypesPage__left">
        {/* <CollectionTypes selectedContentType={selectedContentType} setSelectedContentType={setSelectedContentType} /> */}
        <CollectionTypes />
      </div>
      <div className="contentTypesPage__right">
        <ContentTypes />
        {/* <ContentTypes selectedContentType={selectedContentType} setSelectedContentType={setSelectedContentType} /> */}
      </div>
    </div>
  );
};

export default ContentTypesPage;
