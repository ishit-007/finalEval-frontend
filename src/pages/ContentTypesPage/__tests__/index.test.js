import ContentTypesPage from '..';
import renderer from 'react-test-renderer';
import React from 'react';

jest.mock('react-router-dom');

describe('ContentTypesPage for Login', () => {
  it('should render the ContentTypesPage component', async () => {
    const tree = renderer.create(<ContentTypesPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});