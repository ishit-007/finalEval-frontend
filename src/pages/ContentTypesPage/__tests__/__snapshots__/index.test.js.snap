// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentTypesPage for Login should render the ContentTypesPage component 1`] = `
<div
  className="content-typs-page"
>
  <div
    className="collectionTypesPage__left"
  >
    <div
      className="collection-types"
    >
      <div
        className="heading"
      >
        <h1>
          CMS+
        </h1>
      </div>
      <div
        className="collection-list"
      >
        <div
          className="collection-types-header"
        >
          <h2>
            COLLECTION TYPES
          </h2>
          <img
            alt=""
            src="icon-search-dark.png"
          />
        </div>
        <div
          className="collection-types-list"
        >
          <ul
            className="collection-types-list"
          />
        </div>
        <div
          className="builder"
        >
          <p
            onClick={[Function]}
          >
            CONTENT TYPE BUILDER
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="contentTypesPage__right"
  >
    <div
      className="content-types"
    >
      <div
        className="heading"
      >
        <h1
          onClick={[Function]}
        >
          Content Types
        </h1>
      </div>
      <div
        className="modal"
        style={
          Object {
            "display": "none",
          }
        }
      >
        <div
          className="modal-content"
        >
          <h3>
            Create a New Content Type
          </h3>
          <div
            className="new-content"
          >
            <p>
              Name of the content type
            </p>
            <input
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
          <div
            className="btns"
          >
            <button
              className="cancel"
              onClick={[Function]}
            >
              Cancel
            </button>
            <button
              className="create"
              onClick={[Function]}
            >
              Create
            </button>
          </div>
        </div>
      </div>
      <div
        className="modal2"
        style={
          Object {
            "display": "none",
          }
        }
      >
        <div
          className="modal-content2"
        >
          <h3>
            Create a New Field
          </h3>
          <div
            className="new-content"
          >
            <p>
              Name of the field
            </p>
            <input
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
          <div
            className="btns"
          >
            <button
              className="cancel"
              onClick={[Function]}
            >
              Cancel
            </button>
            <button
              className="create"
              onClick={[Function]}
            >
              Create
            </button>
          </div>
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="new-types-content"
        >
          <div
            className="new-types-header"
          >
            <p>
              0
               Types
            </p>
            <img
              alt=""
              src="icon-search-dark.png"
            />
          </div>
          <div
            className="new-types"
          >
            <div
              className="addNewType"
              onClick={[Function]}
            >
              + New Type
            </div>
            <div
              className="existing-types"
            />
          </div>
          <div
            className="single-type-fields"
          />
        </div>
        <div
          className="one-type-fields"
        >
          <div
            className="heading"
          >
            <h1 />
            <h3>
               Fields
            </h3>
          </div>
          <div
            className="addField"
            onClick={[Function]}
          >
            Add another field
          </div>
          <div
            className="existing-fields"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
